besspin/cyberPhys/cyberphyslib/cyberphyslib/canlib/canlib.py

```diff
@@ -19,7 +19,7 @@ class CanConstants():
     DEST_BROADCAST = "<broadcast>"
```

Comment:
FYI this seems to be OS dependent - recently I had to explicitly use `10.88.88.255` to broadcast packets correctly. Not sure what is an action item here - perhaps just an explaining note?

Reply:
I suspect an explanatory note would be good, but also having this be configurable in the configuration file might make sense to avoid having to edit it for platform/network configuration oddities.

Reply:
Do you mean something like:
```python
import os
if os.name == `posix`:
# do something here...
```

Reply:
Will address this in another task